lib_xua/api/xua_conf_default.h

```diff
@@ -1659,3 +1659,13 @@ enum USBEndpointNumber_Out
 #ifndef WINUSB_DEVICE_INTERFACE_GUID
 #define WINUSB_DEVICE_INTERFACE_GUID               "{89C14132-D389-4FF7-944E-2E33379BB59D}"
 #endif
+
+#ifdef __XC__
+    #define NULLABLE_CLIENT_INTERFACE(tag, name) client interface tag ?name
+    #define NULLABLE_SERVER_INTERFACE(tag, name) server interface tag ?name
+    #define in_port_t in port
+#else
+    #define NULLABLE_CLIENT_INTERFACE(type, name) unsigned name
+    #define NULLABLE_SERVER_INTERFACE(tag, name) server interface tag ?name
```

Comment:
Is this supposed to be the same as the XC define? I wouldn't expect server interface in non-XC code.

Reply:
Agree. Although I copied the format in xccompat.h so prefer to leave it as is. 

In xccompat.h

/**
 * Macro that expands to an client interface with the specified tag when used
 * in an XC source file and to a compatible type when used in a C/C++ source
 * file.
 */
#ifdef __XC__
#define CLIENT_INTERFACE(tag, name) client interface tag name
#else
#define CLIENT_INTERFACE(type, name) unsigned name
#endif

/**
 * Macro that expands to an client interface with the specified tag when used
 * in an XC source file and to a compatible type when used in a C/C++ source
 * file.
 */
#ifdef __XC__
#define SERVER_INTERFACE(tag, name) server interface tag name
#else
#define SERVER_INTERFACE(type, name) unsigned name
#endif

Reply:
Thanks for catching this!